### PR TITLE
Update copyright year

### PR DESCRIPTION
**Pull request by DokaRus**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -19,7 +19,7 @@ http://rafael.xavier.blog.br
 https://github.com/rxaviers/cldrjs/blob/master/LICENSE-MIT
```
Comment (San4es) [inline NOTICE.txt]: @DokaRus 

This link is broken - https://github.com/rxaviers/cldrjs/blob/master/LICENSE-MIT

Let's fix it in context of this PR?

Comment (DokaRus) [reply NOTICE.txt]: Sure

File: layouts/Desktop/DesktopLayout.html [hunk 2]
```diff
@@ -29,7 +29,11 @@ <h2 data-bind="text: text"></h2>
             <div class="footer-gap"></div>
         </div>
         <footer>
-            <div class="wrapper copyright dx-desktop-layout-copyright">2017&nbsp;&copy;&nbsp;Copyright Text</div>
+            <div class="wrapper copyright dx-desktop-layout-copyright">
+                <script type="text/javascript">
```
Comment (San4es) [inline layouts/Desktop/DesktopLayout.html]: @DokaRus 

`<!-- ko text: new Date().getFullYear() --><!--/ko-->`?

File: layouts/Desktop/DesktopLayout.html [hunk 3]
```diff
@@ -29,7 +29,8 @@ <h2 data-bind="text: text"></h2>
             <div class="footer-gap"></div>
         </div>
         <footer>
-            <div class="wrapper copyright dx-desktop-layout-copyright">2017&nbsp;&copy;&nbsp;Copyright Text</div>
+            <div class="wrapper copyright dx-desktop-layout-copyright"><!-- ko text: new Date().getFullYear() --><!--/ko-->&nbsp;&copy;&nbsp;Copyright Text
+            </div>
```
Comment (San4es) [inline layouts/Desktop/DesktopLayout.html]: Unnecessary change

**Review by San4es — APPROVED** [review]
👍
